packages/checkout-storefront/src/sections/DeliveryMethods/DeliveryMethods.tsx

```diff
@@ -38,26 +43,30 @@ export const DeliveryMethods: React.FC<CommonSectionProps> = ({ collapsed }) =>
       <Divider />
       <div className="section" data-testid="deliveryMethods">
         <Title className="mb-2">{formatMessage(deliveryMethodsMessages.deliveryMethods)}</Title>
-        {!shippingAddress && (
+        {!authenticated && !shippingAddress && (
           <Text>{formatMessage(deliveryMethodsMessages.noShippingAddressMessage)}</Text>
         )}
-        <SelectBoxGroup label={formatMessage(deliveryMethodsLabels.deliveryMethods)}>
-          {shippingMethods?.map(
-            ({ id, name, price, minimumDeliveryDays: min, maximumDeliveryDays: max }) => (
-              <SelectBox key={id} name="selectedMethodId" value={id}>
-                <div className="min-h-12 grow flex flex-col justify-center pointer-events-none">
-                  <div className="flex flex-row justify-between self-stretch items-center">
-                    <Text>{name}</Text>
-                    <Text>{getFormattedMoney(price)}</Text>
+        {authenticated && !shippingAddress && updateState.checkoutShippingUpdate ? (
+          <DeliveryMethodsSkeleton />
+        ) : (
```

Comment:
I'm confused, why are we showing skeleton when you're authenticated?

Reply:
if this part should be visible only when you're authenticated then maybe we should first wrap it in an if and then use the turnary operator, like this:

```ts
{authenticated && (
  updateState.checkoutShippingUpdate ? (
    // ...
  ) : (
    // ...
  )
)}
```

Reply:
Hm, the code you've suggested indicates that both the conditions should be displayed only when authenticated, notice that in the current code the last section will display no matter what so... I don't think so? Maybe naming that 3-part condition would make it clearer?